docs/src/data/globalnav.ts

```diff
@@ -0,0 +1,61 @@
+export const RIGHT_NAV_LINKS = [
+  {
+    type: 'EXTERNAL',
+    label: 'Pricing',
+    url: 'https://aws.amazon.com/amplify/pricing',
+    order: 6,
+  },
+  {
+    type: 'EXTERNAL',
+    label: 'About AWS Amplify',
+    url: 'https://aws.amazon.com/amplify/',
+    order: 7,
+  },
+  {
+    type: 'ICON',
+    label: 'Discord',
+    url: 'https://discord.com/invite/amplify',
+    order: 8,
+    icon: 'DISCORD',
+  },
+  {
+    type: 'ICON',
+    label: 'Twitter',
+    url: 'https://twitter.com/AWSAmplify',
+    order: 9,
+    icon: 'TWITTER',
+  },
+];
+
+export const LEFT_NAV_LINKS = [
```

Comment:
These are all external links too though we don't visually show it. But what do you think about adding a more descriptive accessible label (maybe to be used as an aria-label on the NavMenuLink text element)? 

For instance, 'Docs' might be better understood as 'Amplify JS Docs' so that you know it is more specific to the JS libraries. If you are a SR user and you are just tabbing around via links, getting to 'Docs' is a little ambiguous because you might think you are going to get to the Amplify UI Docs from that link if you're just on the home page. Potentially the same with Learn. Having a somewhat more descriptive label like "Learn, video courses for Amplify" (or something maybe better wordsmith-ed) could help make it clear those are external resources. I think this is still an issue for our site for sighted users because this global nav now looks especially more prominent than our own, but this might be a small improvement for SR  users.

Reply:
Yea, that's a great idea. Will look into this!